meritocracy/backend/rest_test.sh

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env bash
+
+declare TOKEN="${1}"
+declare AUTH="Authorization: Bearer ${TOKEN}"
+declare CONTENT="Content-Type: application/json"
+declare HOST='http://localhost:3000'
```

Comment:
why is this hardcoded?

Reply:
TOKEN is provided as a argument to the script (see your next comment) so I do not see the problem here. AUTH and CONTENT are just convenience variables not likely to change.
I grant you that HOST should be an argument like TOKEN.